docs/services/Twitter.md

```diff
@@ -99,7 +101,59 @@ await TwitterService.Instance.StartUserStreamAsync(async tweet =>
 
 // Stop receiving live tweets and events
 TwitterService.Instance.StopUserStream();
-
+```
+```vb
+' Initialize service
+TwitterService.Instance.Initialize(ConsumerKey.Text, ConsumerSecret.Text, CallbackUri.Text)
+
+' Login to Twitter
+If Not Await TwitterService.Instance.LoginAsync() Then
+    Return
+End If
+
+' Get current user info
+Dim user = Await TwitterService.Instance.GetUserAsync()
+ProfileImage.DataContext = user
+
+' Get user time line
+ListView.ItemsSource = Await TwitterService.Instance.GetUserTimeLineAsync(user.ScreenName, 50)
+
+' Post a tweet
+Await TwitterService.Instance.TweetStatusAsync(TweetText.Text)
+Dim status = New TwitterStatus With {
+    .Message = TweetText.Text,
+
+    ' Optional parameters defined by the Twitter "update" API (they may all be null or false)
+
+    .DisplayCoordinates = True,
+    .InReplyToStatusId = "@ValidAccount",
+    .Latitude = validLatitude,
+    .Longitude = validLongitude,
+    .PlaceId = "df51dec6f4ee2b2c",  ' As defined by Twitter
+    .PossiblySensitive = True,      ' As defined by Twitter (nudity, violence, or medical procedures)
+    .TrimUser = True
+}
+Await TwitterService.Instance.TweetStatusAsync(status)
+
+' Post a tweet with a picture
+Await TwitterService.Instance.TweetStatusAsync(TweetText.Text, stream)
+Await TwitterService.Instance.TweetStatusAsync(status, stream)
+
+' Search for a specific tag
+ListView.ItemsSource = Await TwitterService.Instance.SearchAsync(TagText.Text, 50)
+
+' Open a connection with the stream service in order to receive live tweets and events
+ListView.ItemsSource = _tweets
+Await TwitterService.Instance.StartUserStreamAsync(Async Sub(tweet)
+														Await Dispatcher.RunAsync(CoreDispatcherPriority.Normal, Sub()
```

Comment:
Is this normal to be indented like this?

Reply:
VS auto indents like this. If it's unreadable on the docs site I can try and reformat.

Reply:
This is how it looks like on docs: 
![image](https://user-images.githubusercontent.com/711864/38820378-91002292-4152-11e8-98ab-aacb720f06a3.png)

It's readable, but not ideal.

Reply:
fix submitted